dist-build/android-build.sh

```diff
@@ -18,18 +18,23 @@ if [ ! -f ./configure ]; then
   exit 1
 fi
 
-if [ -z "$TARGET_ARCH" ] || [ -z "$ARCH" ] || [ -z "$HOST_COMPILER" ]; then
+if [ "x$HOST_COMPILER" = 'x' ]; then
   echo "You shouldn't use android-build.sh directly, use android-[arch].sh instead" >&2
   exit 1
 fi
 
-export PREFIX="$(pwd)/libsodium-android-${TARGET_ARCH}"
-export TOOLCHAIN_OS_DIR="$(uname | tr '[:upper:]' '[:lower:]')-x86_64/"
+if [ -z ${PREFIX+x} ]; then export PREFIX="$(pwd)/libsodium-android"; echo "setting default PREFIX=${PREFIX}"; fi
+if [ -z ${LIB_PATH_PREFIX+x} ]; then echo ""; else export LIB_PATH="${LIB_PATH_PREFIX}/${TARGET_ARCH_DIR}"; fi
+if [ -z ${LIB_PATH+x} ]; then export LIB_PATH="${PREFIX}/lib/${TARGET_ARCH_DIR}"; fi
+if [ -z ${INCLUDE_PATH+x} ]; then export INCLUDE_PATH="${PREFIX}/include"; fi
```

Comment:
No need for `if` statements. Use `export INCLUDE_PATH="${INCLUDE_PATH:-${PREFIX}/include}"`.

Reply:
the if statements allow someone to override the defaults to suit them. in android.sh there are commented out lines that someone can modify to do something custom without reaching into the scripts.